electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -324,7 +340,19 @@ def update_production_breakdowns(
                     new_event.sourceType,
                 )
 
-        return production_breakdowns
+        if matching_timestamps_only is False:
+            for existing_event in production_breakdowns.events:
```

Comment:
Same here, could be nice to have an helper function for that

Reply:
Will add that in another PR so this don't require another review.